test/common/upstream/cds_api_impl_test.cc

```diff
@@ -112,19 +115,9 @@ TEST_F(CdsApiImplTest, Basic) {
 
   setup();
 
-  std::string response1_json = R"EOF(
-  {
-    "clusters": [
-    {
-      "name": "cluster1"
-    },
-    {
-      "name": "cluster2"
-    }
-    ]
-  }
-  )EOF";
-
+  std::string response1_json = fmt::sprintf(
```

Comment:
nit: as long as you are here can you const this and below. (two can play this game 😉 )